core/src/main/java/com/alibaba/nacos/core/remote/circuitbreaker/CircuitBreaker.java

```diff
@@ -47,22 +56,24 @@
      * @param  pointName entry point name or class name (TODO: can be modified through Nacos console)
      * @return true when the current request is allowed to continue; false if the request breaks the upper limit
      */
```

Comment:
parameter   'connectionId' should be removed

Reply:
sure